lib/jekyll/theme_builder.rb

```diff
@@ -56,12 +56,27 @@ def write_file(filename, contents)
 
   def create_directories
     mkdir_p(SCAFFOLD_DIRECTORIES)
+    mkdir_p("_sass/#{name}")
   end
 
   def create_starter_files
     %w(page post default).each do |layout|
       write_file("_layouts/#{layout}.html", template("_layouts/#{layout}.html"))
     end
+
+    %w(footer head header).each do |incl|
```

Comment:
I personally don't like `head` and `header` because they cause confusion. Maybe `banner` or `top` for the user-facing heading